src/quoridor/server/server.go

```diff
@@ -89,3 +66,58 @@ func getConfiguration(r *http.Request) (*game.Configuration, error) {
 	}
 	return &conf, nil
 }
+
+func getFence(r *http.Request) (game.Fence, error) {
```

Comment:
You should place all our "request parse methods" in another file than server IMO